cmd/backup.go

```diff
@@ -119,7 +121,7 @@ var backupCmd = &cobra.Command{
 					it, err := client.Get().UseDatabase(db).Read(ctx, collection,
```

Comment:
Both the database and the collection can be dropped between the list and the read attempt, so nil checks are needed for both.

Reply:
Thanks for the tip! I'll probably going to keep this simple for now and go for a more robust implementation when the interface is crystallized for doing these transactionally accurately.

Reply:
Updated the description to say "on quiesced database" for now.